core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -519,6 +542,62 @@ class GroupCoordinator(val brokerId: Int,
     groupErrors
   }
 
+  def handleDeleteOffsets(groupId: String, partitions: Seq[TopicPartition]): (Errors, Map[TopicPartition, Errors]) = {
```

Comment:
Leaving this here for lack of a better alternative. One thing that I didn't cover in the KIP is changes to the consumer group command to use this API. I think we can extend the KIP and do this in a separate PR. What do you think?

Reply:
It makes sense. Let's do it in a separate PR.